pkg/cmd/openshift-controller-manager/controller_manager.go

```diff
@@ -74,40 +74,54 @@ func RunOpenShiftControllerManager(config *openshiftcontrolplanev1.OpenShiftCont
 		controllerContext.StartInformers(ctx.Done())
 	}
 
-	eventBroadcaster := record.NewBroadcaster()
-	eventBroadcaster.StartLogging(klog.Infof)
-	eventBroadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
-	eventRecorder := eventBroadcaster.NewRecorder(legacyscheme.Scheme, v1.EventSource{Component: "openshift-controller-manager"})
 	id, err := os.Hostname()
 	if err != nil {
 		return err
 	}
-	rl, err := resourcelock.New(
-		"configmaps",
-		"openshift-controller-manager",
-		"openshift-master-controllers", // this matches what ansible used to set
-		kubeClient.CoreV1(),
-		kubeClient.CoordinationV1(),
-		resourcelock.ResourceLockConfig{
-			Identity:      id,
-			EventRecorder: eventRecorder,
-		})
-	if err != nil {
-		return err
-	}
-	go leaderelection.RunOrDie(context.Background(),
-		leaderelection.LeaderElectionConfig{
-			Lock:          rl,
-			LeaseDuration: config.LeaderElection.LeaseDuration.Duration,
-			RenewDeadline: config.LeaderElection.RenewDeadline.Duration,
-			RetryPeriod:   config.LeaderElection.RetryPeriod.Duration,
-			Callbacks: leaderelection.LeaderCallbacks{
-				OnStartedLeading: originControllerManager,
-				OnStoppedLeading: func() {
-					klog.Fatalf("leaderelection lost")
+	components := []string{"openshift-controller-manager", "cluster-policy-controller"}
```

Comment:
no.  Only do a second leader election on the SCC namespace controller in particular, not all of it.

Reply:
ok

Reply:
I've updated, i'm new to this code, hopefully it's close, with cluster-admin rbac (will figure that out next)  i get the leader lock cm in cluster-kube-controller-manager-ns.